source/crud/update.txt

```diff
@@ -71,8 +71,8 @@ document that matches a specified filter.
         $set: {
           plot: "One month every year, five highly competitive friends
```

Comment:
[suggestion] I can't comment on the correct line, but the movie title specified in line 69 doesn't seem to exist in the sample_mflix collection! Maybe we should swap the information (title and plot) with a movie that _is_ in the collection.